app/src/main/java/org/mozilla/focus/widget/DownloadListAdapter.java

```diff
@@ -88,6 +88,8 @@ private void delete(int position){
     private void cancel(int position){
         DownloadManager manager = (DownloadManager) mContext.getSystemService(Context.DOWNLOAD_SERVICE);
         manager.remove(mDownloadInfo.get(position).getDownloadId());
+
+        remove(position);
```

Comment:
This only removes items canceled inside out app, but not items via Notifications.